automatic/paint.net/tools/ChocolateyInstall.ps1

```diff
@@ -1,20 +1,27 @@
-﻿$ErrorActionPreference = 'Stop'
+$ErrorActionPreference = 'Stop'
 
-$toolsPath = Split-Path $MyInvocation.MyCommand.Definition
+$toolsPath = Split-Path -parent $MyInvocation.MyCommand.Definition
 
 $packageArgs = @{
-  PackageName    = $env:ChocolateyPackageName
-  fileType       = 'exe'
-  file64         = Get-Item $toolsPath\*.exe
-  silentArgs     = '/auto'
+  packageName = $env:ChocolateyPackageName
+  file        = gi $toolsPath\*_x32.zip
+  file64      = gi $toolsPath\*_x64.zip
+  destination = $toolsPath
+  softwareName = 'Paint.NET*'
+  fileType  = 'msi'
+  silentArgs = "/quiet /norestart /l*v `"$($env:TEMP)\$($env:chocolateyPackageName).$($env:chocolateyPackageVersion).MsiInstall.log`""
   validExitCodes = @(0, 1641, 3010)
-  softwareName   = 'Paint.NET*'
 }
 
+Get-ChocolateyUnzip @packageArgs
+Remove-Item $toolsPath\*.zip -ea 0
+
+$FileFullPath = Get-ChildItem $toolsPath -Recurse -Include *.msi | Sort-Object -Descending | Select-Object -First 1
+$packageArgs.Remove('file64')
+$packageArgs['file'] = $FileFullPath
+
 Install-ChocolateyInstallPackage @packageArgs
-Get-ChildItem $toolsPath\*.exe | ForEach-Object { Remove-Item $_ -ea 0; if (Test-Path $_) { Set-Content "$_.ignore" '' }}
 
-$packageName = $packageArgs.packageName
 $installLocation = Get-AppInstallLocation $packageArgs.softwareName
 if (!$installLocation)  { Write-Warning "Can't find $packageName install location"; return }
 Write-Host "$packageName installed to '$installLocation'"
```

Comment:
I don't feel it's a good idea to have this `return` here, as it's going to exit the script at this point. If additional code is added at the end later, this might be missed. And if / else is better here as it's clearer what is happening.